lib/bin/lwm2m_carrier/include/lwm2m_os.h

```diff
@@ -54,7 +54,7 @@ struct lwm2m_os_at_param_list {
 /**
  * @brief AT Command handler.
  */
-typedef void (*lwm2m_os_at_cmd_handler_t)(void *ctx, char *response);
```

Comment:
Not sure you can change this file without release a new `liblwm2m_carrier` library.